app/models/candidate.rb

```diff
@@ -2,7 +2,7 @@ class Candidate < ApplicationRecord
   # Only Devise's :timeoutable module is enabled to handle session expiry
   # Custom Warden strategy is used instead see app/warden/magic_link_token.rb
   devise :timeoutable
-  validates :email_address, presence: true, uniqueness: true, length: { maximum: 250 }
+  validates :email_address, presence: true, length: { maximum: 250 }
```

Comment:
Hmm, why are we removing this? I can't figure out what will go wrong if we leave it in.

Reply:
Sorry, I forgot to add to the description of the PR this. From https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/248/commits/5487d8a2011aeb53914c819cc401849fbd83f094:

>This commit also removes the validation of email uniqueness from Candidate in order to
prevent the 'Email address is already in use' error when the sign up form is used for
an existing email.

**However, I am wrong!** I wrongly assumed (and believed I tested) that having the uniqueness would mean you couldn't continue onwards because the 'Email address is already in use' error would stop you.

Thank you for questioning, I shall put this back! 😅